browser_extension/annotations_to_nanopub.js

```diff
@@ -15,7 +16,7 @@
 // ==/UserScript==
 (async function() {
 
-    const AUTH_TOKEN = "Bearer a1aaab0e-971a-40fe-b7d7-24b40b35a45a";
+    const AUTH_TOKEN = "Bearer aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa";
```

Comment:
How about adding a bit of extra clarification about this token?
```suggestion
    /* this is a hard-coded token to circumvent authentication and Nanopub key generation */
    const AUTH_TOKEN = "Bearer aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa";
```